tests/acceptance/01_vars/04_containers/multiple_cmerge_variable_dynamic.cf

```diff
@@ -0,0 +1,144 @@
+# This policy is synthetic, but represents a real life pattern where data is
+# published from multiple bundles and then merged before use.
+# Note: This has a nearly identical test which uses less variables.
```

Comment:
> $ diff -u multiple_cmerge_variable_static.cf multiple_cmerge_variable_dynamic.cf
--- multiple_cmerge_variable_static.cf	2016-07-26 13:36:33.820902433 -0500
+++ multiple_cmerge_variable_dynamic.cf	2016-07-26 13:36:33.820902433 -0500
@@ -1,5 +1,6 @@
 # This policy is synthetic, but represents a real life pattern where data is
 # published from multiple bundles and then merged before use.
+# Note: This has a nearly identical test which uses less variables.
 body common control
 {
       inputs => { "../../default.cf.sub" };
@@ -35,6 +36,10 @@
         string => "Test that this real life pattern for loading data based on a
                    bundle name and merging it for later use works.";
 
+      "test_soft_fail"
+        string => "any",
+        meta => { "Zendesk#2966" };
+
   methods:
 
       "cron"
@@ -97,7 +102,7 @@
 
   vars:
       "cron"
-        data => readjson( "$(this.promise_filename).policy_1.cron.json"),
+        data => readjson( "$(this.promise_filename).$(this.bundle).$(this.promiser).json"),
         meta => { "cron_rules" };
 
        "s" string => format( "%S", cron );
@@ -120,7 +125,7 @@
       # # For example: Cron entries
        "cron"
 
-         data => readjson( "$(this.promise_filename).policy_2.cron.json"),
+         data => readjson( "$(this.promise_filename).$(this.bundle).$(this.promsier).json"),
          meta => { "cron_rules" };
 
        "s" string => format( "%S", cron );